setup.py

```diff
@@ -83,7 +83,7 @@ def run(self):
     "pygtrie==2.3.2",
     "dpath>=2.0.1,<3",
     "shtab>=1.3.4,<2",
-    "rich>=9.0.0,<10",
+    "rich>=9.0.0",
```

Comment:
This might not be compatible with `rich 9.X` or is it?

Reply:
That sounds right! Will fix.